rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1208,6 +1309,12 @@ def run():
     collection_tube_parser.add_argument("--sample-override", help="for testing",
                                         default=False, action="store_true")  # noqa
 
+    # Backfill tool for genomic_set_member.aw1_file_processed_ID
+    backfill_file_processed_id_parser = subparser.add_parser("file-processed-id-backfill")
+    backfill_file_processed_id_parser.add_argument("--csv", help="A CSV file with the package_ids to backfill",
+                                        default=None, required=True)
+    backfill_file_processed_id_parser.add_argument("--dryrun",
+                                                   help="for testing", default=False, action="store_true")  # noqa
```

Comment:
Everything looks good. Just one possible suggestion: would it make sense to add the "--dry-run" argument on the main parser (`parser`) so it doesn't have to be defined for each subparser?

Reply:
Good suggestion. I've moved it to the main parser and removed each instance from the subparsers.